Plots/Tables/NCL_table_2.py

```diff
@@ -0,0 +1,103 @@
+"""
+NCL_table_2.py
+===============
+This script illustrates the following concepts:
+   - Drawing a table with headers
+   - Filling table cells with a given color
+   - Specifying the position of individual tables
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/table_2.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/table_2_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import matplotlib.pyplot as plt
+
+###############################################################################
+# Plot:
+
+# Create figure and axis
+fig, ax = plt.subplots()
+
+# Set axis aspect ratio
+ax.set_aspect(2)
+
+# Make axis spines invisible
+fig.patch.set_visible(False)
+ax.axis('off')
+
+# Set row headers (first column)
+rows = ["", "", "SLP_ERA40", "Tsfc_ERA40", "Prc_GPCP", "Prc 30S-30N_GPCP",
```

Comment:
How about moving the initialization of variables `rows, rowcolors, cell_text, colors` into a new cell in between "import packages" and "plot" cells and calling it "generate data"?